src/NzbDrone.Api/Calendar/CalendarFeedModule.cs

```diff
@@ -62,6 +67,12 @@ private Response GetCalendarFeed()
                 premiersOnly = bool.Parse(queryPremiersOnly.Value);
             }
 
+            if (queryTags.HasValue)
+            {
+                var tagInput = (string)queryTags.Value.ToString();
+                tags = tagInput.Split(',').Select(_tagService.GetTag).OfType<Tag>().Select(t => t.Id).ToArray();
```

Comment:
And... you just filtered out every event if tags is empty. :smile:

But no worries, I already tweaked the code to be more self-describing.